example/lfp/lfp.cpp

```diff
@@ -260,10 +259,12 @@ int main(int argc, char** argv) {
     std::vector<std::vector<std::array<double, 3>>> samples;
     for (unsigned branch = 0; branch<cell_morphology.num_branches(); ++branch) {
         samples.push_back({});
-        auto branch_range = cell_morphology.branch_indexes(branch);
-        for (auto i_ptr = branch_range.first; i_ptr!=branch_range.second; ++i_ptr) {
-            arb::msample s = cell_morphology.samples()[*i_ptr];
-            samples.back().push_back(std::array<double, 3>{s.loc.x, s.loc.z, s.loc.radius});
+        auto& segments = cell_morphology.branch_segments(branch);
+        auto loc = segments[0].prox;
+        samples.back().push_back(std::array<double, 3>{loc.x, loc.z, loc.radius});
+        for (auto& seg: segments) {
+            auto loc = seg.dist;
+            samples.back().push_back(std::array<double, 3>{loc.x, loc.z, loc.radius});
```

Comment:
Where is the radius used to calculate the LFP? Is it because we have to scale per-area currents to calculate the discrete source terms of the LFP?

I make the change, but will this be sufficient to handle detached segments, in the general case?

Reply:
Yep, this should be fine for general case. This bit here is just to emit geometry for visualization: it doesn't affect the computation of LFP.